src/frontend/src/pages/PPQPage/PPQPage.scss

```diff
@@ -14,7 +14,18 @@
   width: 80%;
   background-color: white;
   padding: 0;
-  padding-left: 10rem;
+  padding-left: 5rem;
+}
+
+.ppq-content {
+  padding: 25px;
+  box-sizing: border-box;
+  margin-right: 4em;
+  background: rgba(59, 153, 252, 0.05);
+}
+
+.ppq-home-warning-icon {
+  color: #fcba19;
```

Comment:
Can we create a color variable?  This will start to create inconsistencies in UI/UX elements.

Reply:
fix it